<?xml version="1.0" encoding="ISO-8859-15"?>

<launch>
  <node name="stereo_image_proc" pkg="stereo_image_proc" type="stereo_image_proc" output="screen">
    <remap from="left/image_raw" to="/sync/cam0/image_raw"/>
    <remap from="left/camera_info" to="/sync/cam0/camera_info"/>
    <remap from="right/image_raw" to="/sync/cam1/image_raw"/>
    <remap from="right/camera_info" to="/sync/cam1/camera_info" />
  </node>

</launch>
